plugins/modules/vmware_object_role_permission.py

```diff
@@ -76,6 +76,9 @@
 EXAMPLES = '''
```

Comment:
```suggestion
EXAMPLES = r'''
```

Please modify this line to start each block using 'r(raw)'.  
Can you please the DOCUMENTATION block the same modifying, too?